TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementCheckers.cs

```diff
@@ -29,6 +29,7 @@ internal static class SetAssignmentStatementChecker
         {
             foreach (var assignment in setAssignmentStatement.Assignments)
             {
+                if (assignment == null) continue;
```

Comment:
Did you actually encounter null assignments ? For which syntax ?

We should not add null values in our lists, otherwise all clients (checkers and future classes) will have to check for null values. Besides, the builder should create as much as possible to capture all the syntax elements. So please check which syntax lead to the null value(s) and adapt the builder consequently.